src/libraries/System.Private.Xml/src/System/Xml/Xsl/Xslt/Scripts.cs

```diff
@@ -41,5 +43,44 @@ public Scripts(Compiler compiler)
             }
             return null;
         }
+
+        internal class LinkerSafeDictionary : IDictionary<string, Type?>
```

Comment:
This works - but do we really need to implement the full `IDictionary`? This class is used in internal code only, so we could probably get by with only the necessary minimum, right?

Ideally I would like to avoid the suppressions for IL2093/IL2092.